torchani/utils.py

```diff
@@ -144,16 +144,16 @@ class EnergyShifter(torch.nn.Module):
             be in order, i.e. ``self_energies[i]`` should be atom type ``i``.
```

Comment:
Could you also add a doc for `fit_intercept` here?

Reply:
Done!